bin/ruby-build

```diff
@@ -1119,13 +1133,27 @@ build_package_mac_openssl() {
   # gives precedence to the last -j option, so we can override that.
   package_option openssl make -j 1
 
-  # Use install_sw instead of install to skip building docs which is slow
-  MAKE_INSTALL_TARGET=install_sw build_package_standard "$@"
+  if is_mac; then
+    # Use install_sw instead of install to skip building docs which is slow
+    MAKE_INSTALL_TARGET=install_sw build_package_standard "$@"
+  else
+    build_package_standard "$@"
+  fi
 
-  # Extract root certs from the system keychain in .pem format and rehash.
-  local pem_file="$OPENSSLDIR/cert.pem"
-  security find-certificate -a -p /Library/Keychains/System.keychain > "$pem_file"
-  security find-certificate -a -p /System/Library/Keychains/SystemRootCertificates.keychain >> "$pem_file"
+  if is_mac; then
+    # Extract root certs from the system keychain in .pem format and rehash.
+    local pem_file="$OPENSSLDIR/cert.pem"
+    security find-certificate -a -p /Library/Keychains/System.keychain > "$pem_file"
+    security find-certificate -a -p /System/Library/Keychains/SystemRootCertificates.keychain >> "$pem_file"
+  else
+    # symlink to the system openssl certs
+    local SYSTEM_OPENSSLDIR
+    SYSTEM_OPENSSLDIR=$(openssl version -d | sed -e 's/.*: "//' -e 's/"$//' || true)
+    if [ -n "$SYSTEM_OPENSSLDIR" ]; then
+      ln -sf "$SYSTEM_OPENSSLDIR/cert.pem" "$OPENSSLDIR/cert.pem"
+      ln -snf "$SYSTEM_OPENSSLDIR/certs" "$OPENSSLDIR/certs"
+    fi
```

Comment:
Could you compare/unify with the logic at https://github.com/rbenv/ruby-build/compare/master...eregon:openssl-linux ?
IIRC there is no cert.pem on Debian-based distributions (at the time I wrote that).